### PR TITLE
Update Extravagant Chocolate Gateau description

### PR DESCRIPTION
**Pull request by mloccy**
[OBJECTS][CATERING]

## About the PR 
Changes the description of the Extravagant Chocolate Gateau to fit in better with the game today and not unfairly shit on countries for being less well off than other countries.

## Why's this needed? 
Doesn't feel like it fits into the game today, but perhaps others will disagree.

### CODE REVIEW

**Comment by klushy225** [comment]
yeah- I think this is a good change. More in-universe references are always good

File: code/modules/food_and_drink/cakes.dm [hunk 1]
```diff
@@ -589,7 +589,7 @@
 
 /obj/item/reagent_containers/food/snacks/cake/chocolate/gateau
 	name = "Extravagant Chocolate Gateau"
-	desc = "Holy shit! This cake probably costs more than the gross domestic product of Bulgaria!"
+	desc = "Holy shit! This cake probably costs more than the peak market cap of the Syndicate!"
```
Comment (Tarmunora) [inline code/modules/food_and_drink/cakes.dm:592]: "The Syndicate" is a fictional terrorist/paramilitary/(etc.) organization bankrolled by corporations with a bone to pick with NT, not a corporation itself.

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drink/cakes.dm (hunk 1 above)
Comment (NibChocolateeny) [reply code/modules/food_and_drink/cakes.dm:592]: https://wiki.ss13.co/Corporations 
@klushy225 maybe this is useful? Donk Co. instead? other crimiminal companies?

Comment (mloccy) [reply code/modules/food_and_drink/cakes.dm]: Updated to Donk Co. Other alternatives could be: "... probably costs more than this station!", "... probably costs more than your organs!". Etc

**Pull request merged**